aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -280,6 +280,12 @@ public class AwsProperties implements Serializable {
    */
   public static final String CLIENT_ASSUME_ROLE_REGION = "client.assume-role.region";
 
+  /**
+   * The value must be one of {@link software.amazon.awssdk.regions.Region}, such as 'us-east-1'.
+   * For more details, see https://docs.aws.amazon.com/general/latest/gr/rande.html
+   */
+  public static final String CLIENT_REGION = "client.region";
```

Comment:
Why `client.region` and not just `region`?

Reply:
@rdblue , @amogh-jahagirdar I'm ok with `client`.
But I wonder is there any cases `s3`/`glue`/`dymano` in a catalog are located in different region.
So, in that case, we need different key for each service like `s3.region`, `glue.region` and `dymano.region`.

Reply:
I decided to create a key for each service: `s3.region`, `glue.region`, `dymano.region` and `kms.region`.